src/graphql/data/fetchGraphQL.ts

```diff
@@ -1,10 +1,10 @@
 import { Variables } from 'react-relay'
-import { CacheConfig, GraphQLResponse, RequestParameters } from 'relay-runtime'
+import { GraphQLResponse, RequestParameters } from 'relay-runtime'
 
-const URL = process.env.REACT_APP_AWS_API_ENDPOINT
+const TOKEN_URL = process.env.REACT_APP_AWS_API_ENDPOINT
```

Comment:
make sense for now but we will eventually merge these endpoints right?

Reply:
Yes